package.json

```diff
@@ -31,6 +32,7 @@
     "cover": "lerna run cover",
     "depcheck": "node scripts/check-mismatched-dependencies.js",
     "docs": "typedoc",
+    "typedoc-markdown": "typedoc --plugin typedoc-plugin-markdown",
```

Comment:
When would this be run? 

Does it need to run at different times than the existing `docs` script?

Should it be in CI to prevent errors like the one you find in `types.js`?

Reply:
This would actually be run in Agoric/documentation repo, which then publish the artifacts to docs.agoric.com via Cloudflare pages.

There should be CI to prevent errors like this, the challenge is that running `yarn typedoc-markdown` itself wouldn't surface the error, it's when we convert `.md` files to html, css, etc using vitepress in `Agoric/documentation` repo where errors will surface. I've summarized where we're at now and what still need to be done in https://github.com/Agoric/documentation/issues/1046

Reply:
I’d be happy if we simulate the whole workflow in our CI so we can see the markdown errors before we merge them.

Reply:
> This would actually be run in Agoric/documentation repo

Seems like an important thing to document in this repo. Perhaps, `docs:markdown-for-documentation-repo`